authproto/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 set(Protobuf_USE_STATIC_LIBS ON)
 
+# XXX(acasey23): Can we revisit this when we get to conan 2? This feels like a bug in the protobuf conan package
+set(CMAKE_PROGRAM_PATH ${CMAKE_PROGRAM_PATH} ${CONAN_BIN_DIRS})
```

Comment:
The cmake code executed when find_package(Protobuf) is run tries to call find_program(protoc ...) but the correct bin directory isn't on it's path for protobuf.

Updating this cache variable fixes the issue for now since find_program reads it to determine where to search